src/nvim/eval.c

```diff
@@ -13042,9 +13047,24 @@ static void f_pyeval(typval_T *argvars, typval_T *rettv, FunPtr fptr)
  */
 static void f_py3eval(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 {
+  if (p_pyx == 0) {
+      p_pyx = 3;
+  }
+
   script_host_eval("python3", argvars, rettv);
 }
 
+// "pyxeval()" function
+static void f_pyxeval(typval_T *argvars, typval_T *rettv)
```

Comment:
This is missing the `FunPtr fptr` parameter, which is why the build is broken...